src/mixins/filing-template-mixin.ts

```diff
@@ -186,7 +188,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
       }
 
       // *** FUTURE: update logic as needed (eg, only save if changed)
-      if (filing.correction.startDate) {
+      if (filing.correction.startDate && this.isEntityTypeFirm) {
```

Comment:
I think this doesn't apply to all firm filings. You might need to identify the filing types this applies to. See also line 184.

Reply:
As this is part of the buildCorrectionFiling method I thought I wouldn't have to filter out what type of filing is needed, and I would only need filter based on the type of entity?

Reply:
Oh I see.

Yes, you're right.

And actually, you don't have to check if this is a firm because line 184 already does that (corrected Registrations or corrected Change of Registrations are firm-only). At some point (future) I may rename those getters to denote that, eg, isCorrectedFirmRegistration.

Reply:
Right okay that makes sense

Reply:
Done